app/src/main/java/com/github/fribourgsdp/radio/FirestoreDatabase.kt

```diff
@@ -192,12 +197,12 @@ class FirestoreDatabase(var refMake: FirestoreRef) : Database {
             }
 
             val id = snapshot.getLong(keyID)!!
-            val nextId = (id + 1) % snapshot.getLong(keyMax)!!
+            val nextId = (id + number) % snapshot.getLong(keyMax)!!
 
             transaction.update(docRef, keyID, nextId)
 
             // Success
-            id
+            Pair(id,nextId)

```

Comment:
I haven't found where the nextId element is used. Is it necessary to have this Pair? Is it to keep track of which id we're on?